noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java

```diff
@@ -119,6 +119,14 @@
 
     Map<String, Integer> sourceFieldNames = new HashMap<>();
 
+    public List<Integer> freq_lvl = List.of(63, 125, 250, 500, 1000, 2000, 4000, 8000);
```

Comment:
Use
```java
Arrays.asList(PropagationProcessPathData.asOctaveBands(PropagationProcessPathData.DEFAULT_FREQUENCIES_THIRD_OCTAVE))
```

Reply:
As the user may have sound source in third octave band or special frequencies bands

Reply:
Done, thanks!

Reply:
Done, thanks!